tests/Unit/Service/ItemServiceTest.php

```diff
@@ -36,6 +37,11 @@ class ItemServiceTest extends TestCase
      * @var \PHPUnit\Framework\MockObject\MockObject|ItemMapper
      */
     private $mapper;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|ItemMapperV2
+     */
+    private $newMapper;
```

Comment:
It would probably be better if you rename the old one to 'oldMapper', since it will probably be removed at some point, and therefore all of them would have to be renamed again. Also, my first thought was that `newsMapper` is meant.